srcpkgs/ardor/template

```diff
@@ -10,7 +10,7 @@ maintainer="Fabio Krapohl <Krapohl.f@gmx.de>"
 license="Jelurida Public License 1.1"
```

Comment:
license="custom:Jelurida Public License 1.1"
https://github.com/void-linux/void-packages/blob/master/Manual.md#mandatory-variables

Reply:
Thanks, I'll add it when I'm on my other machine again.

Reply:
fixed now